postman-collection/IoT Central.postman_collection.json

```diff
@@ -2014,6 +2014,366 @@
 						}
 					},
 					"response": []
+				},
+				{
+					"name": "Add a enrollment group",
```

Comment:
Hi @dominicbetts, I added the changes here for the enrollment groups

Reply:
Sorry - just figured out where the changes are.